packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -61,7 +61,13 @@ export const RequestLineEditFormLayout = ({
       <Grid item xs={4}>
         {Left}
       </Grid>
-      <Grid item xs={4}>
+      <Grid
```

Comment:
to push the input fields down the bottom. I thought it looked better there, now that the heading has been removed.

Reply:
Or vertically centered? I dunno, it's a pretty trivial different tbh.

<img width="1078" alt="Screen Shot 2022-04-12 at 11 21 02 AM" src="https://user-images.githubusercontent.com/5456533/162848949-254da241-d7d8-4803-bea7-91f0e5f983ad.png">

Reply:
yeah, I though that vertically centred would look good.. I thought bottom aligned was the best of the options tried though. The mis-alignment of those input fields isn't great